src/System.Globalization.Extensions/src/System.Globalization.Extensions.csproj

```diff
@@ -79,6 +60,9 @@
     <TargetingPackReference Include="System" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\System.Runtime\pkg\System.Runtime.pkgproj" />
```

Comment:
This should have `<!-- ToDo: Remove this P2P reference once new packages are produced and updated -->`